### ISSUE TITLE
Dismissing modal views by "touch outside"

### ISSUE THREAD

**Issue by yazid** [TeehanLax/UIViewController-Transitions-Example#2]
Referring to the modal view presented when clicking on the cell, is it currently possible to dismiss the same view by touching anywhere outside of the visible area? Not an issue per se, but wondering whether this has been explored with the new API. This is technically the first time that you can officially present a modal view from another UIViewController on an iPhone, so I'm not quite sure how to approach this. Thanks!

**Comment by ashfurrow** [comment]
Yeah, you should be able to add a `UITouchGestureRecognizer` to the container view to respond to touches outside the box. Let me know if you have any questions!

**Comment by Vitaliyy-jomedia** [comment]
Someone has already implemented hide a modal window by clicking on the darkened background? Show me how you did it.
